frontends/web/src/routes/account/account.tsx

```diff
@@ -229,6 +229,7 @@ export function Account({
             </div>
             {isAccountEmpty && (
               <BuyReceiveCTA
+                account={account}
```

Comment:
> this is the only relevant change for this "fix" / "feature".

That it is "the only relevant change" is important info. The other changes are not relevant to the fix.

You could amend to the commit message, but I think it would be much nicer to just have a 2nd commit (same PR) for boyscounting stuff.

Reply:
sorry I should have written that I found the comments very useful ❤️ ... https://github.com/shonsirsha/bitbox-wallet-app/pull/5